docs/HACKING.md

```diff
@@ -133,22 +133,25 @@ To build distribution packages for a specific distribution and release without
 building an actual image, the following command can be used:
 
 ```sh
-mkosi -d <distribution> -r <release> -t none -f
+mkosi -d <distribution> -r <release> -t none
```

Comment:
why drop the "-f"? shouldn't it be at least mentioned somewhere, that it's what you want to specify on rebuilds?

Reply:
`mkosi -t none` is special in that you don't need the `-f` for a rebuild, all it does is rerun the build scripts without ever touching the output directory. With https://github.com/systemd/mkosi/pull/3153 we'll actually start ignoring `-f` when  `-t none` is specified.